src/app/core/services/platform/v1/spender/spender.service copy.ts

```
@@ -0,0 +1,33 @@
+
+  constructor(private httpClient: HttpClient, private dateService: DateService) {
+    this.ROOT_ENDPOINT = environment.ROOT_URL;
+  }
+
+  setRoot(rootUrl: string): void {
+    this.ROOT_ENDPOINT = rootUrl;
+  }
+
+  get<T>(url: string, config = {}): Observable<T> {
+    return this.httpClient
+      .get<T>(this.ROOT_ENDPOINT + '/platform/v1/spender' + url, config)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)